src/simpleTrianglePatterns/simpleTrianglePattern.java

```diff
@@ -0,0 +1,20 @@
+package simpleTrianglePatterns;
+
+import java.util.Scanner;
+public class simpleTrianglePattern {
+    public static void main(String[] args){
+        Scanner scan = new Scanner(System.in);
+        System.out.print("Enter a number:");
+        try{
+            int n = scan.nextInt();
+            for(int i = 1; i <= n; i++){
+                for(int j = 1; j <= i; j++){
+                    System.out.print("*");
+                }
+                System.out.println();
+            }
+        } catch (Exception e) {
+            System.out.println("Wrong format! Please use numbers to enter the input!");
```

Comment:
If i were you, I would have just written that "Wrong format! Please use numbers" but its just personal preference